week-1/errors/3.js

```diff
@@ -1,7 +1,10 @@
-const cardNumber = 4533787178994213;
+const cardNumber = "4533787178994213";
```

Comment:
Whilst this works, I don’t think this is the approach which we were looking for. Have a look at the link below, and consider using method chaining. The missing code should be placed in line 2 before you call the `.slice()` function

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toString#using_tostring